javascript/packages/orchestrator/src/configGenerator.ts

```diff
@@ -9,7 +9,6 @@ import {
 } from "@zombienet/utils";
 import {
   ARGS_TO_REMOVE,
-  DEFAULT_ADDER_COLLATOR_BIN,
```

Comment:
I think you need to import `DEFAULT_ADDER_COLLATOR_BIN` to use it in the check.

Reply:
That was a erroneous merge that should not have happened. Thank you for noticing!